docs/html/_static/pygments.css

```diff
@@ -0,0 +1,77 @@
+.highlight .hll { background-color: #ffffcc }
```

Comment:
do all these javascript and css files have to be in our repo?  They seem to be somewhat boilerplate?  I guess they do as they define our style etc?

Reply:
Yeah, I guess this should be in the repo. (I tried to removed them and the website crashed)